tests/ci/docker_images_check.py

```diff
@@ -110,15 +110,15 @@ def get_changed_docker_images(
                 changed_images.append(DockerImage(dockerfile_dir, name, only_amd64))
                 break
 
-    # Rebuild all images on push or release
-    if pr_info.number == 0:
+    # Rebuild all images on push, release, or scheduled run
+    if pr_info.number in [0,1]:
```

Comment:
This requires a comment about those special values. Why 0 and why 1? The best would be to move those values out as a named constants, like (somewhere in pr_info.py maybe?):
```python
# Since we do both release and scheduled runs on the branch (there is no PR),
# and PR number is an essential part of the CI/CD scripts,
# for simplicity and consistency throughout the codebase,
# we hardcore those runs corresponding pseudo PR numbers:
RELEASE_RUN_PR_ID=0
SCHEDULED_RUN_PR_ID=1
```